library/test/library_test.cpp

```diff
@@ -914,20 +914,33 @@ TEST_F(integration_test, blocking_response_test)
 
 #if 0 // these tests need to be run manually with an active swarm
       // and need the node id to be set below to the first swarm member
-
+#if 0
```

Comment:
normally  I would disable tests by prefixing their name with DISABLED_ so they at least continue to compile.

Reply:
That's fair, I can change that. It was just a convenience as I use these tests a lot for general swarm testing.

Reply:
No need to recompile then, just pass: --gtest_also_run_disabled_tests